### DISCUSSION TITLE
SplitHttp Inbound

### DISCUSSION THREAD

**Discussion by Argo160** [Gozargah/Marzban#1379] [General]
Hello,
In the Xray Inbounds section on Gozargah documents for all SplitHttp examples the following lines have been set which only needs to be set on client sides!! am i wrong?

"maxUploadSize": 1000000,
"maxConcurrentUploads": 10

**Comment by ImMohammad20000** [comment]
We set this and read them from there for subscriptions

**Reply by Argo160** [reply]
> We set this and read them from there for subscriptions

Right
Then what about these as they needs to be set for clients:

"scMaxEachPostBytes": "10000-50000",
"scMaxConcurrentPosts": "50-100",
"scMinPostsIntervalMs": "10-20",
"xPaddingBytes": "50-500",
